### PR TITLE
My_solutions_for_exercise_8

### PR DESCRIPTION
**Pull request by kingsedem**
Here are my solutions to exercise 8. **[](https://expansion.serenity-dojo.com/courses/take/serenity-dojo-java-for-testers/texts/37326337-exercises )**

Kindly note that the comments were cloned from the exercise by default to give more perspective to each task.

### CODE REVIEW

File: src/test/java/com/serenitydojo/WhenWorkingWithSets.java [hunk 1]
```diff
@@ -14,59 +14,69 @@
...
         colors.add("blue");
 
+
         // TODO: Change this code so that it correctly determines whether the set now empty or not
-        Boolean endedUpEmpty = null;
+
+        Boolean endedUpEmpty = colors.isEmpty();
 
         assertThat(startedEmpty).isTrue();
-        assertThat(endedUpEmpty).isFalse();
```
Comment (globalworming) [inline src/test/java/com/serenitydojo/WhenWorkingWithSets.java:69]: why did you remove that line?

**Review by globalworming — APPROVED** [review]
looks quite straight forward :+1:

File: src/test/java/com/serenitydojo/WhenWorkingWithSets.java (hunk 1 above)
Comment (kingsedem) [reply src/test/java/com/serenitydojo/WhenWorkingWithSets.java:69]: oh, sorry it was a mistake, I will re-add before merging. thanks.
